### PR TITLE
use config file when populating our internal list of products with asm

### PR DESCRIPTION
**Pull request by pinzart90**
### Purpose

Make it easier for DEVs and sanbox users to specify in what products Dynamo should look for ASM
Fix https://jira.autodesk.com/browse/DYN-4257

### Release Notes

Use config file when populating our internal list of products with asm

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/Tools/DynamoShapeManager/App.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <appSettings>
+        <add key="productsWithASM" value="Revit,Civil,Robot Structural Analysis,FormIt"/>
```
Comment (pinzart90) [inline src/Tools/DynamoShapeManager/App.config:4]: Should I add ALIAS here too ?
Or should we leave it to DEVs/Sandbox users to specify more products

Comment (mjkkirschner) [reply src/Tools/DynamoShapeManager/App.config:4]: if this is intended for 2.13 my vote is don't add it. If it's just for 2.14 then go for it.

**Comment by mjkkirschner** [comment]
Should this config file be documented somewhere? Maybe https://github.com/DynamoDS/Dynamo/wiki/How-to-Utilize-Dynamo-Builds

**Review by mjkkirschner — APPROVED** [review]
I think this is a good improvement that will make debugging problems with ASM loading a bunch easier.

**Pull request merged**